app/subscriber/src/features/settings/MyMinisterSettings.tsx

```diff
@@ -31,30 +33,58 @@ export const MyMinisterSettings: React.FC = () => {
     if (userInfo?.preferences?.myMinisters) {
       setMyMinisters(userInfo?.preferences?.myMinisters);
     }
-  }, [userInfo, ministers]);
+
```

Comment:
this just checks to see if the current ministers they have saved in their preferences are still active and if any of the ministers have changed status, then they are automatically removed from their preferences withoyt the user having to take any action